### ISSUE TITLE
links in lists not working

### ISSUE THREAD

**Issue by enthus1ast** [soasme/nim-markdown#14]
```markdown
1. [foo](https://nim-lang.org)
2. [baa](https://nim-lang.org/installation)

- [foo](https://nim-lang.org)
- [baa](https://nim-lang.org/installation)
```
get rendered like so:

```
1. [foo](https://nim-lang.org)
2. [baa](https://nim-lang.org/installation)

- [foo](https://nim-lang.org)
- [baa](https://nim-lang.org/installation)
```

but hyperlinks should be created.

**Comment by soasme** [comment]
Thanks for the reporting. This is a known issue and will be addressed in the next release.

**Comment by soasme** [comment]
The bug is fixed in v0.5.0. Plus, the given case has been added into [tests/testmarkdown.nim](https://github.com/soasme/nim-markdown/blob/v0.5.0/tests/testmarkdown.nim#L212-L242).
